tools/st2_deploy.sh

```diff
@@ -147,6 +147,9 @@ create_user() {
     then
       echo "${SYSTEMUSER}    ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers.d/st2
     fi
+    # make sure requiretty is disabled.
+    # unable to get ^Defaults[\s]+requiretty to work.
+    sed -i "s/^Defaults    requiretty/# Defaults    requiretty/g" /etc/sudoers
```

Comment:
Probably safer to use a regex for whitespace (\s+) in case there are more or less whitespace characters.